package.json

```diff
@@ -41,6 +41,12 @@
     "fuzzysort": "^2.0.4",
     "lodash": "^4.17.21",
     "react": "^17.0.2",
+    "prosemirror-view": "1.33.6",
```

Comment:
Are we sure that belong into styleguide?

Reply:
I was trying to avoid it, but since it's used internally by the component and not related to the main editor, I think it's OK.

Reply:
I mean the entire rich editor. Just looking at this changes it seems to be just another thing with no better place where to put it.

Reply:
Kind of. Since it's a contained component, with no business logic I think the style-guide is the best place for it out of our current repos.

I'm happy to un-archive the title-editor repo and keep it there, but it feels like overkill for this much code.